src/NzbDrone.Common/Http/TooManyRequestsException.cs

```diff
@@ -25,5 +25,11 @@ public TooManyRequestsException(HttpRequest request, HttpResponse response)
                 }
             }
         }
+
+        public TooManyRequestsException(HttpRequest request, HttpResponse response, int retryWait)
```

Comment:
Make this a TimeSpan instead of an int so that we can pass any value and it's obvious in the implementation where it's used as to what it is.